cost-analyzer/values.yaml

```diff
@@ -1176,10 +1176,22 @@ clusterController:
   resources: {}
   affinity: {}
   nodeSelector: {}
-  primaryKubecostURL: ""  # URL for secondary clusters to connect to primary Kubecost (ex: https://kubecost.myorganization.com)
-  kubecostAPIKey: ""  # API Key for secondary clusters to authenticate with primary
-  createClusterControllerSecret: true  # disable if you want to use your own secret manager
+  # When using Cluster Controller from a secondary/agent cluster, you will need to set the following to allow
+  # the secondary to query the Primary cluster API for container right-sizing information.
+  # This method should also be used for the Kubecost Primary when using single-sign-on to use an api key (service account).
+  # These setting are not needed when running Kubecost on a single cluster without single-sign-on.
+  usePrimaryApiForRightSizing: false
```

Comment:
Is there a need for this config? Is it redundant with `primaryKubecostURL`? If `primaryKubecostURL` exists then we know we want to use it for rightSizing?